build/js/intlTelInput.js

```diff
@@ -191,6 +193,14 @@ https://github.com/Bluefieldscom/intl-tel-input.git
             $("<div>", {
                 "class": "arrow"
             }).appendTo(selectedFlag);
+            if (this.options.container) {
+                this.$drop = $("<div>", {
+                    "class": "intl-tel-input iti-container"
+                });
+                $("<div>", {
+                    "class": "flag-dropdown"
+                }).appendTo(this.$drop).append(this.countryList);
```

Comment:
this.countryList is not defined yet. And also if you're going to append it to this div here, then you shouldn't append it to the flagsContainer later as well...